Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN apt-get update && apt-get install -y \
         git \
         unzip \
         g++ \
-        mysql-client \
+        default-mysql-client \
```

Comment:
Just curious, what is the `default` for?

Reply:
Tbh i have no idea 😄 

Check out this pipeline: https://gitlab.com/pehapkari/pehapkari.cz/-/jobs/251206782 there was error `E: Package 'mysql-client' has no installation candidate` and by some googling i found that  `mysql-client` was just meta package and by running `apt-cache search mysql-client` i can find replacement for it, output was few packages and one of them was `default-mysql-client`.

Reply:
It was kind of frustrating, because locally `docker build` command worked without any problems and it failed only on CI and i did not find out why.

Reply:
I see :laughing: